endpoints/get.go

```diff
@@ -50,7 +50,7 @@ func parseUUID(r *http.Request, allowKeys bool) (string, error, int) {
 	if len(id) != 36 && (!allowKeys) {
 		// UUIDs are 36 characters long... so this quick check lets us filter out most invalid
 		// ones before even checking the backend.
-		return "", fmt.Errorf("uuid=%s: invalid uuid lenght", id), http.StatusNotFound
+		return id, fmt.Errorf("invalid uuid lenght"), http.StatusNotFound
```

Comment:
Typo: lenght -> length

Reply:
🤦  I'll learn to spell length one day. I promise

Reply:
LOL. I use a VS Code spell check extension to avoid such things. Although, I'll have you know I'm famous in some circles for my misspelling of "unique".